website/config.yaml

```diff
@@ -66,9 +66,9 @@ params:
         url: 'https://slack.k8s.io/'
         icon: fab fa-slack
         desc: 'Chat with us on Slack in the #aws-provider channel'
-  latest_release_version: v0.21.0
+  latest_release_version: v0.21.1
```

Comment:
Is it enough to just update this file, and the docs will update appropriately?

Reply:
Are you talking about if we just change this file but not add the directory? Unfortunately not.